shell/platform/fuchsia/flutter/fuchsia_intl.cc

```diff
@@ -0,0 +1,126 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "fuchsia_intl.h"
+
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "loop.h"
+#include "rapidjson/document.h"
+#include "rapidjson/stringbuffer.h"
+#include "rapidjson/writer.h"
+#include "runner.h"
+#include "runtime/dart/utils/tempfs.h"
+#include "third_party/icu/source/common/unicode/bytestream.h"
+#include "third_party/icu/source/common/unicode/errorcode.h"
+#include "third_party/icu/source/common/unicode/locid.h"
+#include "third_party/icu/source/common/unicode/strenum.h"
+#include "third_party/icu/source/common/unicode/stringpiece.h"
+#include "third_party/icu/source/common/unicode/uloc.h"
+
+namespace {
+
+using icu::Locale;
+
+// This backfills for the static method icu::Locale::forLanguageTag which is
```

Comment:
We should just update the ICU version ourselves. I did it about a year ago so it is definitely a bit old.

Reply:
OK. But, let me do so in a separate change then.

Reply:
Do you have a preference for the version?  ICU 64 has been released on Oct 3.  64.2 was current before, but it also has a lot of patches that would need to be applied too.